src/components/PastAnal/CandidateSummary.vue

```diff
@@ -0,0 +1,140 @@
+<style lang="scss">
+@import '@/styles/main.scss';
+.voteSum__card {
+  width: 100%;
+  height: 100%;
+  min-height: 114px;
+  @include media-breakpoint-up(md) {
+    min-height: 244px;
+  }
+}
+
+.party-info-base {
+  &__logo--shorten {
+    width: 24px;
+    height: 24px;
+    line-height: 24px;
+  }
+}
+</style>
+<template>
+  <div class="py-6 mb-8 d-flex justify-content-center">
+    <div
+      class="col-12 col-md-4 bg-danger rounded-circle"
+      style="height: 246px; width: 246px"
+    ></div>
+  </div>
+  <div class="row">
+    <div class="col-12 col-md-4">
+      <div class="card">
+        <div class="card-header p-4 bg-secondary text-center">
+          <span class="text-danger">1</span>
+        </div>
+        <div class="card-body p-6">
+          <div class="mb-6">
+            <div class="mb-2 d-flex justify-content-between align-items-center">
+              <h5 class="h5">總統候選人</h5>
+              <div class="d-flex align-items-center">
+                <img
+                  :src="getImageUrl(dummyData.candidate.main.avatar_url)"
+                  alt="avatar"
+                  class="me-1 avatar--sm"
+                />
+                <h5 class="h5">
+                  <span class="text-danger">{{
+                    dummyData.candidate.main.name
+                  }}</span>
+                </h5>
+              </div>
+            </div>
+            <div class="mb-2 d-flex justify-content-between align-items-center">
+              <h5 class="h5">副總統候選人</h5>
+              <div class="d-flex align-items-center">
+                <img
+                  :src="getImageUrl(dummyData.candidate.vice.avatar_url)"
+                  alt="avatar"
+                  class="me-1 avatar--sm"
+                />
+                <h5 class="h5">
+                  <span class="text-danger">{{
+                    dummyData.candidate.vice.name
+                  }}</span>
+                </h5>
+              </div>
+            </div>
+            <div class="mb-2 d-flex justify-content-between align-items-center">
+              <h5 class="h5">得票數</h5>
+              <h5 class="h5">
+                <span class="text-dang">{{
```

Comment:
不重要，但我想這裡應該本來是要放 `.text-danger`

```suggestion
                <span class="text-danger">{{
```